bot/cogs/trivia.py

```diff
@@ -0,0 +1,307 @@
+        description="Do .trivia categories to see avaliable choices."
+    )
+    async def trivia(self, ctx: Context, *, category: str = None) -> None:
+        """Trvia command takes in an optional category and sends embed."""
+        if category:
+            category = category.lower()
+        if category == 'categories':
+            embed = Embed(
+                title="Trivia Categories:",
+                description="""
+                        General
+                        Books or Book
+                        Film
+                        Music
+                        Theatre
+                        Televsion or TV
+                        Video Games or VG
+                        Board Games or BG
+                        Science
+                        Computers or Computer
+                        Math
+                        Mythology or Myths
+                        Geography or Geo
+                        History
+                        Art
+                        Animals or Animal
+                        Comics or Comic
+                        Anime
+                        Cartoons or Cartoon
+                            """,
+                color=EMBED_COLOR
+            )
+            embed.set_footer(
+                text="Do .trivia (category name) to choose a category, categories are NOT case sensitive."
+            )
+            await ctx.send(embed=embed)
+            return
+        elif category == 'general':
+            category = 9
+        elif category == 'book' or category == 'books':
+            category = 10
+        elif category == 'film':
+            category = 11
+        elif category == 'music':
+            category = 12
+        elif category == 'theatre':
+            category = 13
+        elif category == 'television' or category == 'tv':
+            category = 14
+        elif category == 'vg' or category == 'video games':
+            category = 15
+        elif category == 'bg' or category == 'board games':
+            category = 16
+        elif category == 'science':
+            category = 17
+        elif category == 'computer' or category == 'computers':
+            category = 18
+        elif category == 'math':
+            category = 19
+        elif category == 'mythology' or category == 'myth':
+            category = 20
+        elif category == 'sports':
+            category = 21
+        elif category == 'geography' or category == 'geo':
+            category = 22
+        elif category == 'history':
+            category = 23
+        elif category == 'art':
+            category = 25
+        elif category == 'animals' or category == 'animal':
+            category = 27
+        elif category == 'comics' or category == 'comic':
+            category = 29
+        elif category == 'anime':
+            category = 31
+        elif category == 'cartoons' or category == 'cartoon':
+            category = 32
```

Comment:
This is all very messy. I would MUCH prefer seeing a dictionary that has the key/value pairs. I also do not like renaming the variable in this way. It may be better to add a new variable `category_id` that corresponds to the numbers.

The dictionary would look like:
```python
my_dict = {"comics": 29, "anime": 31, "math": 19}
``` 
but it should be sorted. Then you could do: `category_id = category_dict[category]`

Reply:
You can also try fuzzy matching to find the correct key (i.e if someone enters `cartoons`, it could resolve to `cartoon`). This is a very small dict, so any search method should be fine.